package.json

```diff
@@ -26,13 +26,13 @@
         "start": "ng serve dev",
         "clean": "del ./dist",
         "test:aot": "ng build --prod ks-app",
-        "test:format": "./scripts/clang-check.sh",
```

Comment:
We should also remove the clang packages from `package.json`.

Reply:
Will push a change to address this.

Reply:
This has been updated per the discussion.